packages/p2p/src/components/floating-rate/floating-rate.jsx

```diff
@@ -0,0 +1,83 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { InputField } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { localize } from 'Components/i18next';
+import { useStores } from 'Stores';
+import './floating-rate.scss';
+
+const FloatingRate = ({
+    changeHandler,
+    class_name,
```

Comment:
Please use snakeCase for this prop.

Reply:
Will incorporate this

Reply:
This isn't updated @likhith-deriv.

Reply:
The prop you referred to was change_handler? Earlier I had created it as changeHandler, considering that it stores a fn() ref.
Are you talking abt a different property?

Reply:
The prop I've referred to is `class_name` as in line 12.

Reply:
Incorporated